packages/react-devtools-shared/src/devtools/views/Components/SelectedElement.js

```diff
@@ -403,6 +405,51 @@ function InspectedElementView({
             ))}
           </div>
         )}
+
+      {source !== null && (
+        <Source fileName={source.fileName} lineNumber={source.lineNumber} />
+      )}
+    </div>
+  );
+}
+
+// The function below (formatSourceForDisplay) is based on packages/shared/describeComponentFrame.js
+const BEFORE_SLASH_RE = /^(.*)[\\\/]/;
```

Comment:
Could you move this inside formatSourceForDisplay? Regexes are mutable, and I'm always wary of sharing them across function runs.

Reply:
Sure. This follows the precedent in other files (like `describeComponentFrame`) but I don't mind making the change here.